README.md

```diff
@@ -99,6 +111,20 @@ signed by the server certificate.
 
 It requires a server certificate with optional certificate authority certificate for the web server and a certificate authority certificate for the validation of client certificates if the client authorization is not disabled.
 
+### HSM Signing
+
+Using the `hsm` signing options signing is switched to hardware-based signing,
+no private key is required anymore. The signing algorithms are the same.
+
+With `--hsm-module` the path to the HSM shared library is specified. This
+library is specific for your hardware signing module, which should be used for
+signing. For testing, you can use the `softhcm` library (for Unix systems
+this is typically `/usr/lib/softhsm/libsofthsm2.so`)
```

Comment:
If you write "for unix systems", i'd assume you want to include mac in your statement. But apparently (at least for me) that's not the case. But you can easily install softhsm using homebrew I guess https://formulae.brew.sh/formula/softhsm.

Reply:
This is the same lib and I used it on Mac to test, so thats all good I believe

Reply:
I think, Unix-like, or `*NIX` might be more expressive, even though I think it should be clear to most readers that not an actual "original" Unix (there were a lot of different *nix-flavours almost from the beginning of *nix anyhow..) is meant.

Reply:
I'll refer to Linux distributions, because unix-derivates like MacOS use different paths)